app/views/confirmation.scala.html

```diff
@@ -32,7 +32,7 @@
 
     <p id="confirmation-email-message">@messages("confirmation.paragraph.confirmationEmail", confirmation.emailAddress)</p>
     <ul class="list">
-        <li><a id="confirmation-pdf-link" target="_blank" href="@routes.PdfDownloadController.application(confirmation.reference, Some(pdfToken))">@messages("confirmation.download.link.PDF")</a></li>
+        <li><a id="confirmation-pdf-link" target="_blank" href="@routes.ApplicationController.viewApplication(confirmation.reference, Some(pdfToken))">@messages("confirmation.download.link.PDF")</a></li>
```

Comment:
message key needs changing

Reply:
This will need to be confirm with Aimee

Reply:
I've change it to download pdf to align it with the content